.drone.yml

```diff
@@ -12,6 +12,7 @@ steps:
   image: quay.io/coreos/jsonnet-ci
   commands:
   - find . -type f -not -path './vendor/*' \( -name '*.libsonnet' -o -name '*.jsonnet' \) | xargs -L1 jsonnetfmt --test
+  - go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@v0.2.0
```

Comment:
Do we want to add this to a Makefile?

Reply:
We don't have a makefile for this repo yet. I'll introduce one in a separate PR.